gapic-generator-cloud/templates/cloud/wrapper_gem/main.erb

```diff
@@ -0,0 +1,5 @@
+<%- assert_locals gem -%>
```

Comment:
why does this redirect-template exist?

Reply:
The ruby layout requires its content to be a partial. (You can see this structure in the default gapic-generator templates also, `service/client.erb` and others.)